src/tool/confiscan.sh

```diff
@@ -115,6 +115,68 @@ else
     mkdir -p "${output_dir}"
 fi
 
+####################
+# Users and groups #
+####################
+
+passwd_file="/etc/passwd"
+[[ -f "${passwd_file}" ]] || error "passwd file not found." 1
+[[ -r "${passwd_file}" ]] || error "passwd file not readable." 1
+
+groups_file="/etc/group"
+[[ -f "${groups_file}" ]] || error "group file not found." 1
+[[ -r "${groups_file}" ]] || error "group file not readable." 1
+
+[[ -d "${output_dir}/users" ]] || mkdir -p "${output_dir}/users"
+[[ -d "${output_dir}/groups" ]] || mkdir -p "${output_dir}/groups"
+
+printf 'Username,UID,GID,Shell,Home\n' > "${output_dir}/users/users.csv"
+cut -d: -f1 ${passwd_file} | paste -d, - \
+    <(cut -d: -f3 ${passwd_file}) \
+    <(cut -d: -f4 ${passwd_file}) \
+    <(cut -d: -f7 ${passwd_file}) \
+    <(cut -d: -f6 ${passwd_file}) >> \
+    "${output_dir}/users/users.csv"
```

Comment:
It would be cool to sort by UID since system users are generally < 1000 and normal users generally >= 1000.

Reply:
Addressed in: https://github.com/AntonVanAssche/hogent-bachelorproef/pull/12/commits/92cff7273909dd8a3c62ce5ac3c59d90ac438e6b.

Added `sort -t, -n -k 2` at the end before writing it to the file.